scripts/tools/flip_video.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+This script helps to flip videos horizontally to create videos of opposite side.
+
+Usage:
+  flip_video.py --path_in=PATH_IN --path_out=PATH_OUT
+  flip_video.py (-h | --help)
+
+Options:
+  --path_in=PATH_IN     Full path to the videos folder
+  --path_out=PATH_OUT   Full path to save flipped videos
+"""
+
+import ffmpeg
+from docopt import docopt
+from pathlib import Path
+import os
+
+if __name__ == '__main__':
+    args = docopt(__doc__)
+    videos_path_in = Path(args['--path_in'])
+    videos_path_out = Path(args['--path_out'])
+
+    video_suffix = '.mp4'
```

Comment:
Do we need to hard-code this? Can we just detect and preserve the same extension as the original?

Reply:
I think I tried once with `.avi` format but then it did not work in case of training. I don't remember perfectly though.